Index: processor-trace-2.0/libipt/src/posix/init.c
===================================================================
--- processor-trace-2.0.orig/libipt/src/posix/init.c
+++ processor-trace-2.0/libipt/src/posix/init.c
@@ -29,7 +29,7 @@
 #include "pt_ild.h"
 
 
-static void __attribute__((constructor)) init(void)
+void __libipt_init(void)
 {
 	/* Initialize the Intel(R) Processor Trace instruction decoder. */
 	pt_ild_init();
